ocs_ci/helpers/cnv_helpers.py

```diff
@@ -0,0 +1,173 @@
+"""
+Helper functions specific for CNV
+"""
+import os
+import base64
+import logging
+
+from ocs_ci.helpers.helpers import create_unique_resource_name, create_resource
+from ocs_ci.ocs import constants
+from ocs_ci.utility import templating
+from ocs_ci.ocs.cnv.virtual_machine import VirtualMachine
+from ocs_ci.utility.utils import TimeoutSampler
+
+logger = logging.getLogger(__name__)
+
+
+def create_vm(
+    namespace,
+    vm_name=None,
+    existing_data_volume_name=None,
+    ssh=True,
+    vm_dict_path=None,
+):
+    """
+    Create a Virtual Machine (VM) in the specified namespace.
+
+    Args:
+        namespace (str): The namespace in which to create the VM.
+        vm_name (str): Name for the VM. If not provided, a unique name will be generated.
+        existing_data_volume_name (str): Existing DataVolume name to attach to the VM
+        ssh (bool): If set to True, it adds a statically manged public SSH key during the VM creation at the first boot
+        vm_dict_path (str): Path to the VM YAML file
+
+    Returns:
+        vm_obj: The VirtualMachine object
+
+    Raises:
+        CommandFailed: If an error occurs during the creation of the VM
+
+    """
+    vm_dict_path = vm_dict_path if vm_dict_path else constants.CNV_VM_CENTOS_YAML
+    vm_data = templating.load_yaml(vm_dict_path)
+    vm_name = vm_name if vm_name else create_unique_resource_name("test", "vm")
+    # data volume is created with the same vm name
+    vm_data["metadata"]["name"] = vm_name
+    vm_data["metadata"]["namespace"] = namespace
+    vm_data["spec"]["dataVolumeTemplates"][0]["metadata"]["name"] = vm_name
```

Comment:
I noticed you are using datavolume templates which won't be supported in 4.15.0. For 4.15.0 we need the stand alone PVC which is referenced by the vm

Reply:
An example of the stand alone PVC referenced by the vm can be seen here:
https://github.com/kgoldbla/ocm-kubevirt-samples/blob/vm9-application/vm-standalone-pvc-odr-regional/pvc.yaml

Reply:
Done